pkgs/applications/misc/open-pdf-presenter/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, cmake, qt4, pythonPackages }:
+
+stdenv.mkDerivation rec {
+  name = "open-pdf-presenter-2014-09-23-f149308";
```

Comment:
Revision in the end is unnecessary. But add `-git-` between name and date, please.
`open-pdf-presenter-git-2014-09-23`